ReactWindows/ReactNative.Shared/ReactInstanceManager.cs

```diff
@@ -610,10 +627,12 @@ private Task<ReactContext> CreateReactContextFromCachedPackagerBundleAsync(Cance
             catch (OperationCanceledException)
             when (token.IsCancellationRequested)
             {
+                RnLog.Info(ReactConstants.RNW, $"ReactInstanceManager: Creating React context has been canceled");
                 throw;
             }
             catch (Exception ex)
             {
+                RnLog.Error(ReactConstants.RNW, ex, $"ReactInstanceManager: Exception when creating React context: {ex.Message}");
```

Comment:
Minor concern for allocations for FormattableString, but very minor.

Reply:
Yeah, hope it's one of those being massively addressed by the gen 0 collection, or it may be allocated on stack for this kind of usages, even better (but it's true that I don't know).

---
In reply to: [221401812](https://github.com/Microsoft/react-native-windows/pull/1987#discussion_r221401812) [](ancestors = 221401812)